### ISSUE TITLE
"local changes" to `symfony.lock` prevent `git checkout` from working

### ISSUE THREAD

**Issue by RayCulp** [kevinpapst/kimai2#2545]
**Describe the bug**
"local changes" to `symfony.lock` prevent `git checkout 1.14` or `git checkout 1.14.1` from working

**To Reproduce**
1. Execute `git fetch --tags` -> no errors
2. Execute `git checkout 1.14` or `git checkout 1.14.1` -> 
```
error: Your local changes to the following files would be overwritten by checkout:
        symfony.lock
Please, commit your changes or stash them before you can switch branches.
Aborting
```

**Additional context**
 - Kimai version: 1.13

No changes were made intentionally to the local file. If any changes were made, they were most likely made by some automated process.

Comparison of old (on server) and new (from repo) versions of this file seem to show that the only difference is in the "doctrine/deprecations" section. 

![2021-04-30-070](https://user-images.githubusercontent.com/7621330/116687570-b3769580-a9b5-11eb-8815-8b8fe916b74f.png)

Is it safe to git stash the local file?

**Comment by kevinpapst** [comment]
Yes. You can stash it. Or `git reset --hard` if `git status` does not report any other (important) changes.

**Comment by RayCulp** [comment]
No other changes reported. Thanks!

**Comment by mauricebar** [comment]
Hello,
I am not really a Linux person but I do my best to try to use Kimai. In the previous updates I ran I had no issues but now in 2 servers with thes same version 1.13 I also get this error. I ran a git status and this is shown:

HEAD detached at 1.13
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   symfony.lock

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        composer.phar

I am not sure if I should do a "reset --hard" or a stash. Sorry if this is a silly question.

Could you give me a hint?

Thanks and regards,
Mauricio

**Comment by kevinpapst** [comment]
Changes on symfony.lock or composer.lock can normally be ignored completely. 
If you didn't apply any changes on purpose (eg. installed LDAP extension) then `git reset --hard` is fine.

**Comment by mauricebar** [comment]
Hello,

Thanks for replying so fast. I haven't changed anything. We have just used the time tracking as usual. I run the "git reset --hard" and got this:
HEAD is now at 9eb25c41 prevent duplicate if overlapping records are disabled (#2311)

After that I tried to run the update again (git checkout 1.14.3) and got this error:
Previous HEAD position was 9eb25c41 prevent duplicate if overlapping records are disabled (#2311)
HEAD is now at df6bd30d fix calendar totals for empty weeks (#2581)

Now the Kimai service is completely down, showing: The server returned a "500 Internal Server Error".
I do have a checkpoint of the server just before I did this, so I can restore it easily, but not sure what happened there.
